central/controllers/controllers_test.go

```diff
@@ -1,10 +0,0 @@
-package controllers_test
-
-import (
```

Comment:
do you need to import test here?

Reply:
nah, file is deleted... cleaning up the test flow right now so it can find this test package in ci

Reply:
Are you going to add that to this PR or a separate one?

Reply:
this one, tests won't pass otherwise

Reply:
will ping you when i update it